src/pages/operator/InputsContainer.tsx

```diff
@@ -0,0 +1,231 @@
+
+interface IProps {
+  classes: any;
+  activeSection: string;
+  configs: configs;
+  handleSubmit(editedConfigs: stringObject): void;
+}
+
+interface IState {
+  booleans: boolObject;
+  strings: stringObject;
+}
+
+const styles = (): any =>
+  createStyles({
+    card: {
+      minWidth: 275
+      // width: '40%'
+    },
+    cardHeader: {
+      textAlign: "left"
+    },
+    cardActions: {
+      float: "right",
+      paddingRight: 32,
+      paddingBottom: 10
+    },
+    cardHeaderTitle: {
+      fontSize: "2.5rem"
+    },
+    cardContentRoot: {
+      paddingTop: 0
+    },
+    footer: {
+      top: "auto",
+      bottom: 0
+    },
+    toolbar: {
+      alignItems: "center",
+      justifyContent: "space-between"
+    },
+    actionButtons: {
+      margin: "0 auto"
+    },
+    button: {
+      fontSize: "1.4rem",
+      margin: "auto 15px"
+    }
+  });
+
+class InputsContainer extends Component<IProps, IState> {
+  public state = {
+    booleans: {},
+    strings: {}
+  };
+
+  private handleBoolanChange = (
+    event: React.ChangeEvent<HTMLInputElement>,
+    name: string
+  ): void => {
+    let booleans: boolObject = { ...this.state.booleans };
+    booleans[name] = event.currentTarget.checked;
+    this.setState({ booleans });
+  };
+
+  private handleStringChange = (
+    event: React.ChangeEvent<HTMLInputElement>,
+    name: string
+  ): void => {
+    let strings: stringObject = { ...this.state.strings };
+    strings[name] = event.currentTarget.value;
+    this.setState({ strings });
+  };
+
+  private handleCancel = (): void => {
+    let booleans: boolObject = {};
+    let strings: stringObject = {};
+    this.setState({ booleans, strings });
+  };
+
+  private handleSubmit = (): void => {
+    let submitObj: stringObject = {
+      ...this.state.booleans,
+      ...this.state.strings
+    };
+    this.props.handleSubmit(submitObj);
+  };
+
+  private shouldActionsDisabled = (): boolean => {
+    if (
+      isEmptyObject(this.state.booleans) &&
+      isEmptyObject(this.state.strings)
+    ) {
+      // return true;  revert back
+      return false;
+    }
+    return false;
+  };
+
+  public render(): ReactNode {
+    const { classes, activeSection, configs } = this.props;
+    const booleans: boolObject = { ...this.state.booleans };
+    const strings: stringObject = { ...this.state.strings };
+    return (
+      <>
+        <Card className={classes.card}>
+          <CardHeader
+            title={<span>{activeSection} settings</span>}
+            className={classes.cardHeader}
+            classes={{ title: classes.cardHeaderTitle }}
+          ></CardHeader>
+
+          <CardContent classes={{ root: classes.cardContentRoot }}>
+            <List>
+              {configs[activeSection] &&
+                configs[activeSection].map(
+                  (config: anyObject, index: number): ReactNode => {
+                    // ConfigType - Number
+                    if (config.type === 1) {
```

Comment:
Lets use an ENUM instead of a number, so instead of 1 lets say TYPE.NUMBER or something like that